packages/internet-header/cypress/e2e/stickyness.cy.ts

```diff
@@ -2,29 +2,33 @@ import { prepare } from '../support/prepare-story';
 
 describe('stickyness', () => {
   beforeEach(() => {
-    prepare('Internet Header/Header Component', 'Default');
+    prepare('Internet Header/Header', 'Default');
   });
 
   it('should not show header when scrolling when stickyness is none', () => {
     cy.get('swisspost-internet-header').should('be.inViewport');
     cy.changeArg('stickyness', 'none');
     cy.wait(10);
-    cy.scrollTo('bottom');
+    cy.get('.header-story-wrapper').scrollTo('bottom');
     cy.get('swisspost-internet-header').should('not.be.inViewport');
   });
 
   it('should not show header when scrolling down with stickyness minimal, should show header without meta when scrolling up a little', () => {
     cy.changeArg('stickyness', 'minimal');
     cy.get('post-meta-navigation').should('be.inViewport');
-    cy.scrollTo('bottom');
-    cy.get('swisspost-internet-footer').should('be.inViewport');
+    cy.get('.header-story-wrapper').scrollTo('bottom');
+    cy.get('.fake-content:last-of-type').should('be.inViewport');
     cy.get('swisspost-internet-header').should('not.be.inViewport');
     cy.get('post-meta-navigation').should('not.be.inViewport');
-    cy.scrollTo('center');
-    cy.scrollTo(0, -20);
+    cy.get('.header-story-wrapper').scrollTo('center', { duration: 10 });
+    cy.get('.header-story-wrapper').then($el => {
+      const el = $el.get(0); //native DOM element
+      el.scrollTo(0, el.scrollTop - 20);
+    });
```

Comment:
This is not lovely, I agree, but I couldn't find another way. The previous behavior (with scrollTo(0,-20) doesn't seem to work anymore with the latest 13.2.0 cypress (work with 12.5.1). It now starts to scroll from the top and not relatively from the current position. The added `{ duration: 10 }` is here to wait for the component. Without it, the stickyness minimal option is not triggered.

I think we can go like this and revise it if it's too flaky.